### DISCUSSION TITLE
How to detect vertices of a character path

### DISCUSSION THREAD

**Discussion by aitormendez** [pixijs/pixijs#8695] [Q&A] [answered]
Hi, I have to make an animation and I'm evaluating if Pixijs can do it. It is about animating a text. The steps to achieve are:

1) Detect the vertices of the characters
2) Convert vertices to dots
3) Join the dots with strokes

The most important question I have is, can Pixijs detect the coordinates of each vertice of a character?

I mean, the anchor points of the bezier curves of a character that have been converted to curves.

![Captura de pantalla 2022-10-05 a las 9 37 34](https://user-images.githubusercontent.com/2788577/194023903-f3cac06e-f846-45bb-9f72-be478baf5467.png)

I guess not. I've been looking at the PIXI.Text class, but there doesn't seem to be a method that detects the vertices of a character. However, I would like, if possible, to have a confirmation.

**Comment by ivanpopelyshev** [comment, marked as answer]
We do not have embedded font parser inside pixi. Text class uses Canvas 2d context built-in rendering and provides only sizes
